app/views/admin/SkippedContactsView.coffee

```diff
@@ -126,43 +151,21 @@ SkippedContactsComponent = Vue.extend
     'skipped-contact-info': SkippedContactInfo
     'flat-layout': FlatLayout
   created: co.wrap ->
+    store.registerModule('page', storeModule()) # TODO: Bring back automatic registering/unregistering for RootComponents?
     try
       skippedContacts = yield api.skippedContacts.getAll()
       @$store.commit('page/loadContacts', skippedContacts)
       yield skippedContacts.map co.wrap (skippedContact) =>
         userHandle = skippedContact.trialRequest?.applicant
+        return
         return unless userHandle
         user = yield api.users.getByHandle(userHandle)
         @$store.commit('page/addUser', { skippedContact , user })
     catch e
       @$store.commit('addPageError', e)
+  destroyed: ->
+    store.unregisterModule('page')
+    @$store = silentStore
```

Comment:
Ah, whoops, I had just yanked that from RootComponent. Looks like it was defined in that file, not this one. I gotta ask you about the store-clearing stuff anyway.